packages/material/src/controls/material-input.control.tsx

```diff
@@ -52,16 +53,13 @@ export class MaterialInputControl extends Control<ControlProps, ControlState> {
           {computeLabel(label, required)}
         </InputLabel>
         <DispatchField uischema={uischema} schema={schema} path={parentPath} />
-        <FormHelperText
-          error={!isValid}
-          hidden={isValid && isDescriptionHidden(visible, description, this.state.isFocused)}
-        >
-          {!isValid ? errors : description}
+        <FormHelperText error={!isValid}>
+          {!isValid ? errors : showDescription ? description : null}
```

Comment:
Inconsistent with vanilla renderer, I think we need to call `formatErrorMessage` here as well.